packages/web-components/src/toolbar/toolbar.spec.md

```diff
@@ -0,0 +1,56 @@
+# Toolbar
+
+## Component Description
+
+A Toolbar is a container for grouping a set of controls: action controls (e.g. button, icon button), input fields, dropdown, and menu button. Each control will be a web component.
+
+Checkbox too?
+
+The actions and input fields on a toolbar can include dropdowns.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/QfHyx6ApwFEcw5q4pM9CSZ/Toolbar?node-id=2351-6672&t=MuajdFxoCBjwlK4w-0)
+
+[Link to Fluent 2 Spec](https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-toolbar--default)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr appearance: "static" | "contextual"
+- @attr size: "small" "medium" | "medium"
+- @attr vertical: boolean | false
+  -- FAST uses `orientation`: enum { horizontal, vertical }
+
+### Outputs
+
+### Slots
+
+- start: Content which appears at the start of the toolbar, before the default slotted items.
+- default: Content which appears in the default location of the toolbar, left-aligned.
+- end: Content which appears at the end of the toolbar, right aligned.
+
+### CSS Variables
+
+None
+
+## Interactions
+
+None - Interactions such as hover and click are subject to the UX rules of the individual components slotted in the toolbar.
+
+## Accessibility
+
+- [x] Find the matching component through WCAG's patterns: https://www.w3.org/WAI/ARIA/apg/patterns/
+  - https://www.w3.org/WAI/ARIA/apg/patterns/toolbar.html
+- [x] Are there any accessibility elements unique to this component?
+- [x] List ARIA attributes
+  - [x] role="toolbar" in FAST
+  - [x] aria-orientation="vertical" in FAST
+- [ ] Does the component support 400% zoom?
+- [ ] What keyboard behaviors does the component support?
+  - [x] Tab : Moves focus to the next focusable element in the toolbar.
+  - [x] Shift + Tab : Moves focus to the previous focusable element in the toolbar.
+  - [x] Arrow keys : Moves focus to the next/previous focusable element in the toolbar.
+
+## Preparation
```

Comment:
Looks like the template got cut out here. Let's make sure to document the deltas between the web component and React versions for future reference:

Here's the complete Preparation section
## Preparation

- [ ] [Find the base FAST Component](https://explore.fast.design/components/) this component will inherit from and document
  - [FAST Component](https://explore.fast.design/components/fast-text-area)
- [ ] [Check the Fluent UI React V9 Component Spec](https://github.com/microsoft/fluentui/tree/master/specs) for differences and document
  - Difference 1
- [ ] [Fluent UI React V9 Storybook](https://aka.ms/fluentui-storybook) for implementation differences and document
  - Difference 1
- [ ] [Open GitHub issues related to component](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#find-open-issues-on-github)
  - [link to each issue]
- [ ] (Optional) [Draft implementation](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#draft-implementation)
  - [link to draft implementation, if applicable]
- [ ] [Component Spec authored](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#component-spec) and [reviewed](https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#spec-review)